master/reference/calicoctl/resources/globalnetworkpolicy.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Global Network Policy Resource (GlobalNetworkPolicy)
+---
+
+A Global Network Policy resource (GlobalNetworkPolicy) represents an ordered set of rules which are applied
+to a collection of endpoints which match a [label selector](#selector).
+
+GlobalNetworkPolicy is not a namespaced resource, which means, GlobalNetworkPolicy applies to [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint) in all namespaces. 
+See [NetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) for namespaced Network Policy.
```

Comment:
Is there anywhere we can go to discover what "namespace" means, exactly? A search of our docs only turns up the Kubernetes "namespace". Of course, we know there are some "namespaces" in Linux, as well. So, it's hard to know what is a "namespace" in this case.

Reply:
We don't have a page explaining that. Maybe we should add one. Where do you think we could add that? It might just be a 10-20 lines page (only applicable to WEP and NP resources)

Reply:
It might be possible to add this as a new section at the bottom of this section https://docs.projectcalico.org/master/reference/calicoctl/resources/ . We can always move it later. But that will give you a link you can point to as an anchor link (auto-created). So if you added a section called "About namespaces" you could refer to it with a link called `https://docs.projectcalico.org/{{page.version}}/reference/calicoctl/resources/#about-namespaces`. that could be a quick fix solution for the alpha purposes, if you have time.